src/main/java/com/google/devtools/build/lib/query2/output/QueryOptions.java

```diff
@@ -48,6 +50,18 @@ public OrderOutputConverter() {
           + " Allowed values are: label, label_kind, minrank, maxrank, package, location, graph,"
           + " xml, proto, record.")
   public String outputFormat;
+  @Option(name = "null",
```

Comment:
Let's just leave this option, and not specify arbitrary string for line terminator (I believe that is the extent of configurability Unix tools have, and having arbitrary terminators beg questions of whether those terminators are escaped in the output otherwise)